types/src/transaction/authenticator.rs

```diff
@@ -884,6 +886,9 @@ pub enum AnySignature {
     Secp256k1Ecdsa {
         signature: secp256k1_ecdsa::Signature,
     },
+    WebAuthn {
+        partial_authenticator_assertion_response: PartialAuthenticatorAssertionResponse,
```

Comment:
Good question. I think it works better with the mental model for WebAuthn since you're technically verifying the response from an assertion, not just verifying the signature. Additionally so it's helpful to make it more explicit that you're not just verifying the signature, especially since `PartialAuthenticatorAssertionResponse` includes a `signature` field, but happy to reconsider it if you feel strongly

Reply:
Disagree, but not sure if this is a battle I'm gonna win 🤣

Reply:
loooool. Idk i feel like the number of times signature is used, and then reused, and then composed by other signatures is unhealthy :D. But I will change it if you feel strongly

Reply:
Changed to `signature` per @alinush's recommendation 🙏